### PR TITLE
[Security Content] Add Windows Investigation Guides

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

This PR adds Investigation Guides to the following rules:

* First Time Seen Commonly Abused Remote Access Tool Execution
* FirstTime Seen Account Performing DCSync
* PowerShell Invoke-NinjaCopy script
* PowerShell Script with Encryption/Decryption Capabilities
* Process Termination followed by Deletion
* Potential Secure File Deletion via SDelete Utility
* Renamed Utility Executed with Short Program Name (Previously: Suspicious Execution - Short Program Name)
* Suspicious Zoom Child Process
* Enumerating Domain Trusts via DSQUERY.EXE
* Enumerating Domain Trusts via NLTEST.EXE

### CODE REVIEW

File: rules/windows/command_and_control_new_terms_commonly_abused_rat_execution.toml [hunk 1]
```diff
@@ -19,6 +19,40 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
...
+
+This rule detects when a remote access tool is seen in the environment for the first time in the last 15 days, enabling analysts to investigate and enforce the correct usage of such tools.
+
+#### Possible investigation steps
+
+- Investigate the process execution chain (parent process tree) for unknown processes. Examine their executable files for prevalence, whether they are located in expected locations, and if they are signed with valid digital signatures.
+- Check if the execution of the remote access tool is approved by the organization's IT department.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Contact the account owner and confirm whether they are aware of this activity.
+  - If the tool is not approved for use in the organization, the employee could have been tricked into installing it and providing access to a malicious third party. This third party could be attempting to scam the end-user or gain access to the environment through social engineering.
```
Comment (joepeeples) [inline rules/windows/command_and_control_new_terms_commonly_abused_rat_execution.toml:39]: Nit: This isn't exactly a step or action the reader can take; it just describes a possible scenario. Maybe:
```suggestion
  - If the tool is not approved for use in the organization, the employee could have been tricked into installing it and providing access to a malicious third party. Investigate whether this third party could be attempting to scam the end-user or gain access to the environment through social engineering.
```

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 2]
```diff
@@ -18,7 +18,44 @@ index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
...
+
+Active Directory data consists of objects that have properties, or attributes. Each object is an instance of an object class, and object classes and their respective attributes are defined in the Active Directory schema. Objects are defined by the values of their attributes, and changes to attribute values must be transferred from the domain controller on which they occur to every other domain controller that stores a replica of an affected object.
+
+Adversaries can use the DCSync technique that uses Windows Domain Controller's API to simulate the replication process from a remote domain controller, compromising major credential material such as the Kerberos krbtgt keys used legitimately for tickets creation, but also tickets forging by attackers. This attack requires some extended privileges to succeed (DS-Replication-Get-Changes and DS-Replication-Get-Changes-All), which are granted by default to members of the Administrators, Domain Admins, Enterprise Admins, and Domain Controllers groups. Privileged accounts can be abused to grant controlled objects the right to DCsync/Replicate.
```
Comment (joepeeples) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:29]:
```suggestion
Adversaries can use the DCSync technique that uses Windows Domain Controller's API to simulate the replication process from a remote domain controller, compromising major credential material such as the Kerberos krbtgt keys that are used legitimately for creating tickets, but also for forging tickets by attackers. This attack requires some extended privileges to succeed (DS-Replication-Get-Changes and DS-Replication-Get-Changes-All), which are granted by default to members of the Administrators, Domain Admins, Enterprise Admins, and Domain Controllers groups. Privileged accounts can be abused to grant controlled objects the right to DCsync/Replicate.
```

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 3]
```diff
@@ -18,7 +18,44 @@ index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
...
+
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Contact the account and system owners and confirm whether they are aware of this activity.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Correlate security events 4662 and 4624 (Logon Type 3) by their Logon ID (`winlog.logon.id`) on the Domain Controller (DC) that received the replication request. This will tell you where the AD replication request came from, and if it came from another DC or not.
+- Scope which credentials were compromised (for example, whether all accounts were replicated or specific ones).
+
+### False positive analysis
+
+- Administrators may use custom accounts on Azure AD Connect, investigate if this is part of a new Azure AD account setup, and ensure it is properly secured. If the activity was expected and there is no other suspicious activity involving the host or user, the analyst can dismiss the alert.
```
Comment (joepeeples) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:45]:
```suggestion
- Administrators may use custom accounts on Azure AD Connect; investigate if this is part of a new Azure AD account setup, and ensure it is properly secured. If the activity was expected and there is no other suspicious activity involving the host or user, the analyst can dismiss the alert.
```

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 4]
```diff
@@ -18,7 +18,44 @@ index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
...
+- Although replicating Active Directory (AD) data to non-Domain Controllers is not a common practice and is generally not recommended from a security perspective, some software vendors may require it for their products to function correctly. Investigate if this is part of a new product setup, and ensure it is properly secured. If the activity was expected and there is no other suspicious activity involving the host or user, the analyst can dismiss the alert.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are identified. Reset passwords for these accounts and other potentially compromised credentials, such as email, business systems, and web services.
+- If the entire domain or the `krbtgt` user were compromised:
```
Comment (joepeeples) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:52]: (Verb is singular if the compound subject uses `or`.)
```suggestion
- If the entire domain or the `krbtgt` user was compromised:
```

File: rules/windows/credential_access_dcsync_replication_rights.toml [hunk 5]
```diff
@@ -48,8 +48,7 @@ This rule monitors for Event ID 4662 (Operation was performed on an Active Direc
 ### Response and remediation
 
 - Initiate the incident response process based on the outcome of the triage.
-- If specific credentials were compromised:
-  - Reset the password for these accounts and other potentially compromised credentials, like email, business systems, and web services.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are identified. Reset passwords for these accounts and other potentially compromised credentials, such as email, business systems, and web services.
 - If the entire domain or the `krbtgt` user were compromised:
```
Comment (joepeeples) [inline rules/windows/credential_access_dcsync_replication_rights.toml:52]:
```suggestion
- If the entire domain or the `krbtgt` user was compromised:
```

File: rules/windows/defense_evasion_posh_encryption.toml [hunk 6]
```diff
@@ -18,6 +18,36 @@ index = ["winlogbeat-*", "logs-windows.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "PowerShell Script with Encryption/Decryption Capabilities"
+note = """## Triage and analysis
+
+### Investigating PowerShell Script with Encryption/Decryption Capabilities
+
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks, making it available for use in various environments, creating an attractive way for attackers to execute code.
+
+PowerShell offers encryption and decryption functionalities that can be abused by attackers for a variety of purposes, from concealing payloads and C2 communications to bypass security protections to ransomware capabilities.
```
Comment (joepeeples) [inline rules/windows/defense_evasion_posh_encryption.toml:27]: The "from ... to" format gets a little confusing with a long phrase, maybe try something like:
```suggestion
PowerShell offers encryption and decryption functionalities that can be abused by attackers for a variety of purposes, such as concealing payloads and C2 communications to bypass security protections and performing ransomware capabilities.
```

File: rules/windows/defense_evasion_sdelete_like_filename_rename.toml [hunk 7]
```diff
@@ -19,11 +19,35 @@ license = "Elastic License v2"
 name = "Potential Secure File Deletion via SDelete Utility"
 note = """## Triage and analysis
 
-Verify process details such as command line and hash to confirm this activity legitimacy.
+### Investigating Potential Secure File Deletion via SDelete Utility
 
-## Setup
+Sdelete is a tool primarily used for securely deleting data from storage devices, making it unrecoverable. Microsoft develops it as part of the Sysinternals Suite. Although commonly used to delete data securely, attackers can abuse it to delete forensic indicators and remove files as a post-action to a destructive action such as ransomware or data theft to hinder recovery efforts.
```
Comment (joepeeples) [inline rules/windows/defense_evasion_sdelete_like_filename_rename.toml:24]:
```suggestion
SDelete is a tool primarily used for securely deleting data from storage devices, making it unrecoverable. Microsoft develops it as part of the Sysinternals Suite. Although commonly used to delete data securely, attackers can abuse it to delete forensic indicators and remove files as a post-action to a destructive action such as ransomware or data theft to hinder recovery efforts.
```

File: rules/windows/defense_evasion_suspicious_zoom_child_process.toml [hunk 8]
```diff
@@ -17,9 +43,56 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious Zoom Child Process"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Suspicious Zoom Child Process
+
+By examining the specific traits of Windows binaries -- such as process trees, command lines, network connections, registry modifications, and so on -- it's possible to establish a baseline of normal activity. Deviations from this baseline can indicate malicious activity, such as masquerading and deserve further investigation.
```
Comment (joepeeples) [inline rules/windows/defense_evasion_suspicious_zoom_child_process.toml:50]:
```suggestion
By examining the specific traits of Windows binaries -- such as process trees, command lines, network connections, registry modifications, and so on -- it's possible to establish a baseline of normal activity. Deviations from this baseline can indicate malicious activity, such as masquerading, and deserve further investigation.
```

**Review by joepeeples — COMMENTED** [review]
Just a few small suggested edits -- thanks for preparing this content!

FYI, to streamline reviews, the Security Docs team will now have a single tech writer reviewing most PRs for prebuilt rules & investigation guides. No need to wait for reviews from the other writers on this one, thanks!

**Review by imays11 — APPROVED** [review]
Great work

**Review by joepeeples — APPROVED** [review]
LGTM! Thanks again for preparing this content!

**Review by Aegrah — APPROVED** [review]
Very nice work, looks good!

**Pull request merged**